### PR TITLE
Format block height with commas

### PR DESCRIPTION
**Pull request by shubhamkmr04**
https://github.com/ZeusLN/zeus/issues/2602

### CODE REVIEW

**Comment by shubhamkmr04** [comment]
Screenshot_1734082878

@kaloudis should we also format some values here on this view ?

**Comment by kaloudis** [comment]
> Screenshot_1734082878
> 
> @kaloudis should we also format some values here on this view ?

Yes, let's do it

File: views/NetworkInfo.tsx [hunk 1]
```diff
@@ -72,13 +73,17 @@ export default class NetworkInfo extends React.Component<NetworkInfoProps, {}> {
 
                 <FlatList
                     data={NETWORK_INFO}
-                    renderItem={({ item }) => (
+                    renderItem={({ item, index }) => (
                         <KeyValue
                             keyValue={localeString(item.label)}
-                            value={item.value}
+                            value={
+                                index < 3
```
Comment (kaloudis) [inline views/NetworkInfo.tsx:80]: not crazy about using the index here

Comment (shubhamkmr04) [reply views/NetworkInfo.tsx]: pushed few changes! would this work?

**Pull request merged**